webapp/plugins/plugins.js

```diff
@@ -261,6 +261,9 @@ export const decorateTheme = themeCreator => {
 };
 
 export function getPluginReducers() {
+  // return nothing if no plugin reducers
+  if (!pluginReducers.length) return null;
```

Comment:
Its not the most ideal that this references global state

Reply:
That's basically how the plugin build system works though, it uses these as a cache essentially (see the line right below at l.269). I can see how it's not ideal, but this was also the pattern borrowed from hyper)